initfs/scripts/moslo-build.sh

```diff
@@ -58,6 +58,32 @@ debug()
         fi
 }
 
+LIB_PATHS="/usr/lib /lib"
+
+# Function to find _direct_ lib dependencies for executable using objdump
+objdump_find_lib_path()
+{
+        local FOUND_LIBS=""
+        local LIB_LIST=""
+        local FOUND_LIB=""
+
+        if test -z $1; then
+                echo "find_lib_path: Error: Empty executable name!"
+                exit 1
+        fi
+        LIB_LIST=$(objdump -p "$1" | grep NEEDED | sed -e 's/\NEEDED//g')
+        for lib in $LIB_LIST; do
+                for path in $LIB_PATHS; do
+                        FOUND_LIB=$(find $path/ | grep $lib)
```

Comment:
hmm.. I originally had it just "find $path/$lib" but it then occured to me that there might be some libraries under sub-directories... I'll see if I can do something there, thanks.